pkg/utils/metadatas/metadata.go

```diff
@@ -216,6 +216,49 @@ func getFromMetadataService(metadataVersion string) (*Metadata, error) {
 	return parseMetadata(resp.Body)
 }
 
+func GetDevicePath(volumeID string) string {
+	// Nova Hyper-V hosts cannot override disk SCSI IDs. In order to locate
+	// volumes, we're querying the metadata service. Note that the Hyper-V
+	// driver will include device metadata for untagged volumes as well.
+	//
+	// We're avoiding using cached metadata (or the configdrive),
+	// relying on the metadata service.
+	instanceMetadata, err := getFromMetadataService(defaultMetadataVersion)
+
+	if err != nil {
+		klog.V(4).Infof("Could not retrieve instance metadata. Error: %v", err)
```

Comment:
err message should use: klog.Errorf()
use "klog.Errorf()" replace klog.V(4).Infof()

Reply:
nice